### DISCUSSION TITLE
Cmd.exe support

### DISCUSSION THREAD

**Discussion by GameLord2011** [lptstr/winfetch#223] [Ideas]
Maybe you could add support for cmd.exe?

**Comment by rashil2000** [comment]
If you've installed via Scoop or Chocolatey, you can already use it inside Cmd by just running `winfetch`. 
If you've installed via other means, just create a small batch script containing this line:
```batch
 powershell -nop -file "C:\path\to\winfetch\script\winfetch.ps1"  %*
```
and put it somewhere in your PATH.

**Reply by GameLord2011** [reply]
I am using a DOSKEY for winfetch.bat now, and cmd.exe cannot run .ps1 files.

**Reply by rashil2000** [reply]
```batch
doskey winfetch=powershell -nop -file "C:\path\to\winfetch\script\winfetch.ps1"  $*
```
